test/functionalTest/cases/0000_ipv6_support/ipv4_only.test.DISABLED

```diff
@@ -0,0 +1,156 @@
+# Copyright 2013 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Note this .test has been renamed with .DISABLED. Have a look to https://github.com/telefonicaid/fiware-orion/issues/1954 to know the reason.